package.json

```diff
@@ -34,6 +37,11 @@
   },
   "homepage": "https://github.com/mjzone/serverless-boilerplate#readme",
   "devDependencies": {
-    "opn": "4.0.2"
+    "chai": "^4.1.1",
+    "enzyme": "^2.9.1",
+    "mocha": "^3.5.0",
+    "opn": "4.0.2",
+    "react-addons-test-utils": "^15.6.0",
+    "sinon": "^3.2.1"
```

Comment:
@Janakshi28 
How about using `jest` as the testing framework since this is in react. It's has lots of things required for testing react components.

Reply:
@dinukadesilva 
GIven the positive response from the React community, I think that's a good option too. But since there was a recommendation to use Mocha and chai in issue #19 , I thought of going with mocha and chai.